draft-irtf-cfrg-frost.md

```diff
@@ -479,22 +411,54 @@ a single, trusted dealer, and the other which requires performing a distributed
 key generation protocol. We highlight key generation mechanism by a trusted dealer
 in {{dep-dealer}}, for reference.
 
-FROST assumes the existence of a *Coordinator*, which is a Signer responsible for the following:
+There are two rounds in FROST: commitment and signature share generation. The first round serves
+for each participant to issue a commitment. The second round receives commitments for all signers as well
+as the message, and issues a signature share. The Coordinator performs the coordination of each
+of these rounds. At the end of the second round, the Coordinator then performs an aggregation
+round at the end and outputs the final signature. This complete interaction is shown in {{fig-frost}}.
 
-1. Determining which signers will participate (at least THRESHOLD_LIMIT in number);
-2. Coordinating rounds (receiving and forwarding inputs among participants); and
-3. Aggregating signature shares output by each participant, and publishing the resulting signature.
+~~~
+        (group info)             (group info,     (group info,
+            |                     signing key)     signing key)
```

Comment:
This is `signing key share`

Reply:
Copy-pasted from existing text. We can update it.